### PR TITLE
Changes made for wacom_i2c

### PR DESCRIPTION
**Pull request by flying-elephant**
These changes include a modification on Makefile.in and an addition of wacom_i2c driver.
Please review.
Thanks!

### CODE REVIEW

**Comment by Pinglinux** [comment]
A pull request is normally from a branch other than the master of our own repo. Since this is your first time, we'll accept it from your master branch. My comments are in the individual patches. Thank you Tats!

**Review by Pinglinux — COMMENTED** [review]
I assume your wacom_i2c.c was based on the wacom_i2c.c at https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/drivers/input/touchscreen/wacom_i2c.c  as of 2021-11-29. Commit: fafc66387dc0 (Input: wacom_i2c - clean up the query device fields). This information should be included in your commit comments.

Except that, I approve this patch.

File: 4.5/Makefile.in [hunk 1]
```diff
@@ -47,19 +46,18 @@ clean:
...
 install modules_install:
 	$(MAKE) -C $(WCM_KERNEL_DIR) M=$(PWD) modules_install mod_sign_cmd='$(MODSIGN_COMMAND)'
 	mkdir -p /etc/depmod.d
 	echo "override wacom * extra" > /etc/depmod.d/input-wacom.conf
-	echo "override wacom_w8001 * extra" >> /etc/depmod.d/input-wacom.conf
 	PATH="$(PATH):/bin:/sbin" depmod -a $(MODUTS)
 ifdef UPDATE_INITRAMFS
 	$(UPDATE_INITRAMFS) -u -k $(MODUTS)
 endif
-ifdef DRACUT
```
Comment (Pinglinux) [inline 4.5/Makefile.in:60]: @jigpu What do you think of the change in this block?

File: 4.5/Makefile.in [hunk 2]
```diff
@@ -10,12 +10,11 @@ ccflags-y := -DWACOM_VERSION_SUFFIX=\"-$(WCM_VERSION)\" -Wall -Wextra -Wno-unuse
 wacom-objs := wacom_wac.o wacom_sys.o
 obj-m += wacom.o
 obj-m += wacom_w8001.o
+obj-m += wacom_i2c.o
 else
 $(error You requested to build wacom with configure, but wacom is configured as built-in in your kernel config)
 endif # CONFIG_HID_WACOM
 
-LINUXINCLUDE := $(subst -I, -isystem, $(LINUXINCLUDE))
-
```
Comment (Pinglinux) [inline 4.5/Makefile.in:18]: @jigpu Also, is it ok to remove the above line?

File: 4.5/Makefile.in (hunk 1 above)
Comment (flying-elephant) [reply 4.5/Makefile.in:60]: That's accidental deletion, sorry.
I'll send the corrected patch.

File: 4.5/Makefile.in (hunk 2 above)
Comment (flying-elephant) [reply 4.5/Makefile.in:18]: This is too, accidental deletion.
I'm preparing another patch.

**Comment by jigpu** [comment]
Closing with #303 taking its place.

**Pull request closed without merging**